dev/test/document.ts

```diff
@@ -2073,9 +2073,13 @@ describe('update document', () => {
     }).to.throw(INVALID_ARGUMENTS_TO_UPDATE);
 
     expect(() => {
-      firestore
-        .doc('collectionId/documentId')
-        .update({foo: 'bar'}, {exists: true}, 'foo');
+      firestore.doc('collectionId/documentId').update(
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
Would `// @ts-expect-error` work here, instead of `@ts-ignore`, and the corresponding lint check disabling?